source/plugin/workspace/eclipse.rst

```diff
@@ -55,23 +62,40 @@ Maven
 * Select ``Create a simple project``, unless you require a more advanced setup.
 * Enter your **Group ID**, **Artifact ID**, **Version**, project name, and project description.
 
-  * Your **Group ID** should correspond to your organization name, or something similar.
-  * Your **Artifact ID** should correspond to your project name.
+  * Your **Group ID** should usually correspond to your the **plugin group ID** you chose earlier, e.g.
+    ``io.github.username``.
+  * Your **Artifact ID** should usually correspond to your **unqualified plugin ID** you chose earlier, e.g.
+    ``myplugin``.
   * Your **Version** is up to you.
+
 * Click ``Finish``.
 
 .. tip::
 
-    Eclipse often does not open new projects after creating them. If this happens, try closing the Eclipse welcome screen;
-    your project should be open behind it.
+    Eclipse often does not open new projects after creating them. If this happens, try closing the Eclipse welcome
+    screen; your project should be open behind it.
+
+Editing the project configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-**To import the Sponge API as a dependency:**
+.. TODO: How does this work? That wouldn't add Sponge's Maven repository
```

Comment:
I'm rather confused by how these were supposed to work, although I haven't tested it. Still, I think it would be better to actually shortly explain how to use Maven (-> edit the build configuration file), instead of just having them use a generator